auto-value-gson-annotations/build.gradle

```diff
@@ -0,0 +1,5 @@
+apply plugin: 'java'
+
+dependencies {
+    compile 'com.google.code.gson:gson:2.6.2'
+}
```

Comment:
You need to add `apply from: rootProject.file('gradle/gradle-mvn-push.gradle')` to this file.